packages/database/src/client.ts

```diff
@@ -1,21 +1,17 @@
 import { PrismaClient } from "@prisma/client";
 import { withAccelerate } from "@prisma/extension-accelerate";
 
-const prismaClientSingleton = () => {
-  return new PrismaClient({
-    datasources: { db: { url: process.env.DATABASE_URL } },
-    ...(process.env.DEBUG === "1" && {
-      log: ["query", "info"],
-    }),
-  }).$extends(withAccelerate());
-};
-
-type PrismaClientSingleton = ReturnType<typeof prismaClientSingleton>;
+const prismaClientSingleton = new PrismaClient({
```

Comment:
why was this change from a function to a direct assignment needed as the function would be more efficient for the singleton pattern as the prisma client is only created when needed.

Reply:
I actually got a linting error saying that, we need a return type for function `prismaClientSingleton`. When we extend it `withAccelerate`, return type of function changes from `PrismaClient` and I didn't find a type def for it in prisma docs and hence decided to change it to direct assignemnt, but i get you concern here. It is better if its a function 🤔